decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'Nameable'
+require_relative 'Person'
```

Comment:
- kindly make sure you use the same name that you give to your file as is shown above you start with a capital letter but your file name starts with small letter please fix that to well access your files :+1: